test/integ/separableCompilation/src/main.cpp

```diff
@@ -111,23 +111,21 @@ TEMPLATE_LIST_TEST_CASE("separableCompilation", "[separableCompilation]", TestAc
     alpaka::memcpy(queueAcc, memBufAccA, memBufHostA);
     alpaka::memcpy(queueAcc, memBufAccB, memBufHostB);
 
-    auto const& bundeledKernel
-        = alpaka::KernelBundle(kernel, memBufAccA.data(), memBufAccB.data(), memBufAccC.data(), numElements);
+    auto const& kernelBundle = alpaka::KernelBundle(
+        kernel,
+        std::data(memBufAccA),
```

Comment:
I do like `x.data()` more than `std::data(x)` (it's shorter, and the latter does not gain anything), but I guess it's a matter of taste.

Reply:
yes this is matter of tast and was in the code before too.